README.md

```diff
@@ -1 +1,22 @@
-# beyond08-group1
```

Comment:
Please remove this line. We have a real name for our project (Tasties), I don't think putting the temporary group name back in here would give anyone any relevant information.

Reply:
Any update on this?